packages/x-date-pickers-pro/src/internals/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -81,7 +81,7 @@ export const useDesktopRangePicker = <
     ...pickerParams,
     props,
     wrapperVariant: 'desktop',
-    autoFocusView: true,
+    autoFocusView: false,
```

Comment:
This is technically a bug-fix as our range picker behavior is for the focus to always stay on the field.